### PR TITLE
feat: format and cleanup notebooks

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

**Review by kyle-cochran — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ run-and-edit-jupyter: ## Run and edit jupyter notebooks
 		--volume="$(project_directory)/notebooks:/notebooks" \
 		--workdir=/notebooks \
 		$(jupyter_image) \
-		bash -c "start-notebook.py --ServerApp.token=''"
+		bash -c "pip install kaleido~=0.2; start-notebook.py --ServerApp.token=''"
```
Comment (Derollez) [inline Makefile:62]: what's the purpose if this?

Comment (vishwa2710) [reply Makefile:62]: For plotly, so that we can save output as PNG/SVG

Comment (Derollez) [reply Makefile:62]: 🙏

Comment (lucas-bremond) [reply Makefile:62]: It'd odd to install stuff at boot time no? Worth updating the `requirements.txt` instead?

Comment (Derollez) [reply Makefile:62]: Is it because we don't have a dependency on it and just use it for tutorials notebooks?

Comment (vishwa2710) [reply Makefile:62]: Agreed @lucas-bremond . I just wanted to get something out the door. This was already a nerd-snipe XD.

Currently we don't have a requirements.txt, we just use the latest OSTk astro image. We can update this to actually build and use a Docker image from the Jupyter Dockerfile that is in the repo (and is currently unused? from what I can tell).

**Pull request merged**